news/2024/2024-06-17-the-followpoint-dressurf-top-keyboard-smasher

```diff
@@ -0,0 +1,110 @@
+>
+> After finally getting S-rank on that chart, my hand speed and reading abilities improved to new heights. I remember grinding hours and hours to just get a pass on that chart.
+
+*Did you ever see yourself becoming the rank 1 player in osu!mania, why or why not?*
+
+> Not even close, I never saw myself becoming the rank 1 when I was barely passing 6 star charts.
+>
+> Just like every other player, playing 7key genre was hard. Actually, everything was hard. Coordinating fingers to each column, Reading seven keys at once, Playing charts even though I don't feel like improving...
+>
+> Even watching professional players looked so hard and confusing, but I still had my goal to achieve #1 in the future kept going.
+
+*What do you feel like most people don't truly realize about being a top player, especially in the highest ranks?*
+
+> There's undeniable pressure that I always expect and get, Such as setting 1900pp's or trying to maintain current spot... So I always practice off-stream to get in shape, exploring pending tab on webpage, downloading new charts, rating them to play on stream later.
+
+*Who would you say are the best osu!mania player(s) at the moment that aren't you, and why?*
+
+> I think the term "best" is very subjective, if I see "best" in terms of inspiring and drawing players into the osu!mania scene, I must say jhlee0133 for 7key genre, Sillyfangirl for 4k genre.
+
+*Is there any advice that you would give to people who are just starting out, or are wanting to improve to one day be like you?*
+
+> Well, it's going to be really hard to correlate your fingers to each column, but must be preceded first to get used to 7 keys playstyle. Once you become accustomed to the 7-key playstyle and develop muscle memory for finger positioning, your progress will accelerate.
+
+*Do you think that peripherals matter when it comes to becoming better at the game, why or why not?*
+
+> Yes, if you use a keyboard that's high, I strongly recommend utilizing a wrist rest or at least a towel to alleviate potential wrist pain.
+>
+> A minimum requirement for keyboard I think is the ability to press 7+ keys properly at once. Additionally, 144hz monitor would be great advantage if you're going to step up into more extreme charts.
+
+*What drives you to keep improving despite being at the top?*
+
+> Until now, and still, I always set myself as rival and competed against myself, consistently challenging my previous scores to improve. I deliberately make charts' rate a little bit over my playlimit and play them when my hand's condition is very good. Of course, I get 91~92% on it, but I primarily concentrate on getting scores to battle on later.
+
+*What are some "endgame" scores or milestones you would want to achieve in the future?*
+
+> I want to develop at least 300+ bpm chordjack, 270~280+ bpm brackets or passing Stellium regular rice dan on 1.1x.

```

Comment:
```suggestion
> I want to develop atleast 300+ BPM chordjack, 270-280+ BPM brackets or passing Stellium Regular Rice Dan on 1.1x.

```

Reply:
link the dan?

Reply:
Related to this probably https://osu.ppy.sh/beatmapsets/451788#mania/1673133